README.md

```diff
@@ -172,12 +171,6 @@ Known labextensions:
 NOTE: If you don't see the Elyra server extension enabled, you may need to explicitly enable
 it with `jupyter serverextension enable elyra`
 
-## Configuring Runtimes for Pipeline execution
```

Comment:
Why are we removing this?

Reply:
I was trying to address  several structural issues:
- The section sequence "Verify Installation", "Configuring Runtimes for Pipeline execution"  "Starting Elyra" was invalid (one can't "configure a runtime" without "starting Elyra " first).
 - The section was without context and appeared to assume that the reader was already familiar with pipelines. The revised feature summary links to the relevant pipeline documentation, which provides the conceptual overview as well as links to the how-to steps.

If your concern is that we don't feature pipelines more prominently, we could probably add a short subsection to the overview.